### ISSUE TITLE
Question: Is there a recommended way to train for "others" or "none of the above" labels?

### ISSUE THREAD

**Issue by yash98** [Knowledgator/GLiClass#29]
I am training for a multilabel classification task where it's expected to predict no class in some cases

**_Based on your experience is there some suggestion on how to train well for this?_**

Options I am considering -

1. Train with just a selected text string like "others" / "none of the above"
2. Don't make anything for special class, simply setting output logits to 0 vector and using MLP scorer
3. Add a special token to represent "others" / "none of the above" class
